src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -511,7 +511,9 @@ export class TerminalInstance implements ITerminalInstance {
 			this._configHelper.mergeDefaultShellPathAndArgs(shell);
 		}
 		this._initialCwd = this._getCwd(this._shellLaunchConfig, this._historyService.getLastActiveWorkspaceRoot());
-		const env = TerminalInstance.createTerminalEnv(process.env, shell, this._initialCwd, locale, this._cols, this._rows);
+		const platformKey = platform.isWindows ? 'windows' : platform.isMacintosh ? 'osx' : 'linux';
+		const envFromConfig = { ...process.env, ...this._configHelper.config.envVars[platformKey] };
```

Comment:
Will this replace process.env entries with the envVars one when conflicts occur?